frontend/src/static/js/v_summary.js

```diff
@@ -9,9 +9,71 @@ window.comparator = (fn) => function compare(a, b) {
   return 1;
 };
 
+// ui funcs, only allow one ramp to be highlighted //
+let drags = [];
+
+function getBaseTarget(target) {
+  return (target.className === 'summary-chart__ramp')
+    ? target : getBaseTarget(target.parentElement);
```

Comment:
same for here, needs another 2 space of indentation

Reply:
oh yah, these are the one that was complained about in the eslint, i changed them once, then i saw the eslint got error.

Reply:
https://travis-ci.org/reposense/RepoSense/jobs/527111802

Reply:
seems like eslint needs to be updated with this rule then 😛

Reply:
i do that in this pr?

Reply:
can do it towards the master, then fix it by rebase or when merging

Reply:
eh... i checked the memberexpression already is 2, i think it doesn't really recognize the `?` `:` if else clause as a parameter.

the other choice is to make everything one line, but its kinda not very nice to read, i keep the current style? cos it doesn't really make sense to change it to the full if else statement either.

Reply:
that is probably a different rule

Reply:
should be this one https://eslint.org/docs/rules/indent#flatternaryexpressions

Reply:
doesn't do double indent one worh, is it okay if i on this, put it to a new line, but only one indent?

Reply:
can you try `"ConditionalExpression": 2`?

Reply:
![image](https://user-images.githubusercontent.com/1430854/57298529-0853de00-7105-11e9-8cb5-a9e45a073592.png)

this was the only reference to conditionalexpression leh

Reply:
yes, just think that we should test whether if it is an undocumented feature. It doesn't quite make sense to add this to ignore nodes but not to the indentation.

If it doesn't work for indentation, we can add the `ignoreNodes` rule anyway so that it can adhere to our standard.